modules/cnv_cloning_pvc_datavolume.adoc

```diff
@@ -40,7 +40,7 @@ spec:
       - ReadWriteOnce
     resources:
       requests:
-storage: 500Mi
+        storage: 2gi
```

Comment:
The indentation LGTM, but I don't think you needed to change the value to `storage: 2gi`. In 2.0, we still have `storage: 500Mi` for this example (see https://github.com/openshift/openshift-docs/pull/15598/files#diff-e25406d973bae44a7fcc4e45c697f2b6). I think it should be capitalized `Gi` if you're going to change it.